File: python/tests/fizz_buzz_test.py
from slow_code.fizz_buzz import fizz_buzz
import pytest


class TestFizzBuzz:

    @pytest.mark.skip(reason="test currently disabled")
    def test_acceptance_test(self) -> None:
        # /!\ Should only pass at the end:
        # - Keep skipped
        # - Write and pass intermediate tests
        # - Un-skip at the end when should be passing

        assert fizz_buzz(90) == "FizzBuzz"
        assert fizz_buzz(91) == "91"
        assert fizz_buzz(92) == "92"
        assert fizz_buzz(93) == "Fizz"
        assert fizz_buzz(94) == "94"
        assert fizz_buzz(95) == "Buzz"
        assert fizz_buzz(96) == "Fizz"
        assert fizz_buzz(97) == "97"
        assert fizz_buzz(98) == "98"
        assert fizz_buzz(99) == "Fizz"
        assert fizz_buzz(100) == "Buzz"

File: php/tests/FizzBuzzTest.php
<?php
namespace SlowKata\Tests;
use SlowKata\FizzBuzz;
use PHPUnit\Framework\TestCase;

class FizzBuzzTest extends TestCase
{
    /** @test */
    public function AcceptanceTests()
    {
        // /!\ Should only pass at the end:
        // - Keep disabled
        // - Write and pass intermediate tests
        // - Enable at the end when should be passing
        // /!\ IF YOU PASS IT... YOU MIGHT HAVE GONE TOO FAST!
        $this->markTestSkipped('this test could eventually be run once a full implementation has been written');
        $fizzBuzz = new FizzBuzz();
        $this->assertEquals('1', $fizzBuzz->say(1));
        $this->assertEquals('2', $fizzBuzz->say(2));
        $this->assertEquals('Fizz', $fizzBuzz->say(3));
        $this->assertEquals('Buzz', $fizzBuzz->say(5));
        $this->assertEquals('FizzBuzz', $fizzBuzz->say(15));
    }
}